src/v0/destinations/tiktok_ads_offline_events/data/TikTokTrack.json

```diff
@@ -0,0 +1,43 @@
+[
+  {
+    "destKey": "event_set_id",
+    "sourceKeys": "properties.eventSetId",
+    "required": false
+  },
+  {
+    "destKey": "event_id",
+    "sourceKeys": ["properties.eventId", "messageId"],
+    "required": false
```

Comment:
Same as above

Reply:
Although it's mentioned required in the doc, events are flowing successfully without it.